packages/compose/README.md

```diff
@@ -0,0 +1,27 @@
+# @wordpress/compose
+
+The `compose` package is a collection of handy [Higher Order Components](https://facebook.github.io/react/docs/higher-order-components.html) you can use to wrap your WordPress components and provide some basic features like: State, instanceId, pure...
```

Comment:
I think we should use the documentation to make it extremely clear the distinction we have been modules like `compose`, `components`, and `elements` (and `data`, `viewport`, etc), as even internally we've not been entirely consistent with what the approach is. With this being an effort to standardize the approach, we could do well to explain it thoroughly to avoid ambiguity.